src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/CreatePoolControllerITest.java

```diff
@@ -1530,6 +1530,30 @@ public void addCitizensToCoronerPool_courtUser() throws Exception {
 
     }
 
+    @Test
+    @Sql({"/db/mod/truncate.sql", "/db/pool-management/create_voters_disqualified_only.sql",
+        "/db/CreatePoolController_createPool.sql"})
+    public void createPool_NotEnoughJurors_OnlyDisqualified() throws Exception {
+
+        final String bureauJwt = mintBureauJwt(BureauJwtPayload.builder()
+            .userType(UserType.BUREAU)
+            .login("BUREAU_USER")
+            .staff(BureauJwtPayload.Staff.builder().name("Bureau User").active(1).build())
+            .owner("400")
+            .build());
+
+        PoolCreateRequestDto poolCreateRequest = setUpPoolCreateRequestDto();
+        poolCreateRequest.setCitizensToSummon(1);
+
+        final URI uri = URI.create("/api/v1/moj/pool-create/create-pool");
+
+        httpHeaders.set(HttpHeaders.AUTHORIZATION, bureauJwt);
+        RequestEntity<PoolCreateRequestDto> requestEntity = new RequestEntity<>(poolCreateRequest, httpHeaders,
+            HttpMethod.POST, uri);
+        ResponseEntity<String> response = template.exchange(requestEntity, String.class);
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
I would expect a BVR here instead of an INTERNAL_SERVER_ERROR

Reply:
That's existing logic of the create pool controller that it throws a runtime exception, would update but would impact front-end as it will need changing to handle the new error (in context to this endpoint).

Reply:
Just updated this from the discussion earlier this morning to use BVR

Reply:
Sorry this is my miss understanding this should be an INTERNAL_SERVER_ERROR as it is chucked inside a try/catch block on any unexpected exception.

(I thought it was business logic checking from the test name)